packages/common/src/scrollgrid/SimpleScrollGrid.tsx

```diff
@@ -193,10 +193,20 @@ export class SimpleScrollGrid extends BaseComponent<SimpleScrollGridProps, Simpl
 
   // TODO: can do a really simple print-view. dont need to join rows
   handleSizing = () => {
-    this.setState({
-      shrinkWidth: this.computeShrinkWidth(), // will create each chunk's <colgroup>. TODO: precompute hasShrinkWidth
-      ...this.computeScrollerDims(),
-    })
+    const nextShrinkWidth = this.computeShrinkWidth()
+    const nextDims = this.computeScrollerDims()
+
+    if (
+      this.state.shrinkWidth !== nextShrinkWidth ||
+      this.state.forceYScrollbars !== nextDims.forceYScrollbars ||
+      JSON.stringify(this.state.scrollerClientHeights) !== JSON.stringify(nextDims.scrollerClientHeights) ||
+      JSON.stringify(this.state.scrollerClientWidths) !== JSON.stringify(nextDims.scrollerClientWidths)
```

Comment:
I think you can use `isPropsEqual` instead to compare the scroller width and height. Using `JSON.stringify` to compare is not the best for performance 🙂 
```suggestion
      this.state.shrinkWidth !== nextShrinkWidth
      || this.state.forceYScrollbars !== nextDims.forceYScrollbars
      || isPropsEqual(this.state.scrollerClientHeights, nextDims.scrollerClientHeights)
      || isPropsEqual(this.state.scrollerClientWidths, nextDims.scrollerClientWidths)
```

Reply:
I didn't even notice it was imported. Good call! These objects are not huge, but if anyone adds more properties to these objects it could lead to an annoying bug otherwise.